docs/building-spark.md

```diff
@@ -83,13 +83,10 @@ Example:
 
 To enable Hive integration for Spark SQL along with its JDBC server and CLI,
 add the `-Phive` and `-Phive-thriftserver` profiles to your existing build options.
-By default, Spark will use Hive 1.2.1 with the `hadoop-2.7` profile, and Hive 2.3.6 with the `hadoop-3.2` profile.
-
-    # With Hive 1.2.1 support
-    ./build/mvn -Pyarn -Phive -Phive-thriftserver -DskipTests clean package
+By default Spark will build with Hive 2.3.6.
 
     # With Hive 2.3.6 support
-    ./build/mvn -Pyarn -Phive -Phive-thriftserver -Phadoop-3.2 -DskipTests clean package
+    ./build/mvn -Pyarn -Phive -Phive-thriftserver -DskipTests clean package
```

Comment:
Shall we add the Hive 1.2.1 example back after this for users?
```
# With Hive 1.2.1 support
./build/mvn -Pyarn -Phive -Phive-thriftserver -Phive-1.2 -DskipTests clean package
```

Reply:
cc @gatorsmile and @srowen

Reply:
OK either way. I guess we kind of want to discourage Hive 1.x, but doesn't mean it should have a line in the docs still